prefect_aws/s3bucket.py

```diff
@@ -0,0 +1,80 @@
+import io
+from typing import Optional, Union
+from uuid import uuid4
+
+import boto3
+from anyio import to_thread
+from prefect.filesystems import ReadableFileSystem, WritableFileSystem
+
+from prefect_aws import AwsCredentials, MinIOCredentials
+
+
+class S3Bucket(ReadableFileSystem, WritableFileSystem):
+
+    """
+    Block used to store data using S3-compatible object storage like MinIO.
+
+    Args:
+        bucket_name: Name of your bucket.
+        credentials: A block containing your credentials (AwsCredentials or
+        MinIOCredentials).
```

Comment:
Indent for new line of same kwarg
```suggestion
            MinIOCredentials).
```